test/e2e/authentication/authentication.page.ts

```diff
@@ -13,27 +13,27 @@ export class AuthenticationPageObject {
 
   }
 
-  openApp(){
+  public openApp(): webdriver.promise.Promise<void> {
```

Comment:
Hi Sam, yes indeed you could. But you could also omit the return type or type it as any.

To me, this particular notation really highlights how you intended the method to be used.